lxd/patches.go

```diff
@@ -1568,4 +1569,30 @@ func patchMoveBackupsImagesStorage(name string, d *Daemon) error {
 	return nil
 }
 
+// patchClusterConfigVolatileUUID checks if the clusterUUID is defined and if not, adds the server UUID to config under
+// 'volatile.uuid'. Note that this means for existing deployments (i.e. for whom this patch applies) the cluster UUID is
+// a v4 UUID, whereas new installations will have a v7 UUID.
+func patchClusterConfigVolatileUUID(name string, d *Daemon) error {
```

Comment:
@markylaing have you considered that in addition to:

> Note that this means for existing deployments (i.e. for whom this patch applies) the cluster UUID is
// a v4 UUID, whereas new installations will have a v7 UUID.

That the UUID will not necessarily be the Node UUID associated with the cluster certificate.

I wonder if for both of those reasons we just instead generate a new random v7 uuid for existing deployments.

What do you think?

Reply:
At least that way its always a v7